pkg/util/uint160_test.go

```diff
@@ -5,75 +5,69 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestUint160UnmarshalJSON(t *testing.T) {
 	str := "2d3b96ae1bcc5a585e075e3b81920210dec16302"
 	expected, err := Uint160DecodeString(str)
-	if err != nil {
-		t.Fatal(err)
-	}
+	assert.NoError(t, err)
```

Comment:
You're mixing adding tests and refactoring old tests in a single 39b51bb68441f509f33bd154a7189710ca350291 commit, that's just not good, split them into two separate commits, it makes reviewing them and referring to them easier.